### DISCUSSION TITLE
Add more helpers for QInput `rules`

### DISCUSSION THREAD

**Discussion by mathiasayivor** [quasarframework/quasar#14822] [General - Components / Directives / etc]
Are there plans to add more helpers to the QInput `rules` list?
Currently, there are only a few helpers, meaning the developer is likely to duplicate input validation logic per each project manually.
IMO, validation is done on most inputs, hence basic validation rules should be added to make basic input validation easier and faster

**Comment by MilosPaunovic** [comment]
What rules have you had in mind? PRs are always welcome 🙂

**Reply by mathiasayivor** [reply]
Basic validation rules like `required`, `minLength`, `maxLength`, etc.

**Reply by metalsadman** [reply]
there's vuelidate or vee-validate.

**Reply by mathiasayivor** [reply]
Ikr, but it means the developer would have to duplicate code per each project (setting up vuelidate/vee-validate for validating inputs).
Most projects do basic input validation (they have at least a `required` input field) and it would be helpful to add such rules to the existing basic validation rules provided by quasar to cater for such basic validation needs instead of installing another library such as vuelidate/vee-validate.
